README.md

```diff
@@ -54,11 +37,17 @@ Varies depending on operating system, package manager and Ruby installation. Gen
 	[sudo] gem install gollum
 	```
 
-Installation examples for individual systems can be seen [here](https://github.com/gollum/gollum/wiki/Installation).
+To run, simply:
+
+1. Navigate to your git repository (wiki) via the command line.
+2. Run: `gollum`.
+3. Open `http://127.0.0.1:4567` in your browser.
+
+Of course, you can also run `gollum /path/to/wiki`. Installation examples for individual systems can be seen [here](https://github.com/gollum/gollum/wiki/Installation).
```

Comment:
Perhaps this should be the default recommendation. When I started using gollum I was annoyed to find that all my pages had cluttered my gollum install.